lightning/src/ln/payment_tests.rs

```diff
@@ -351,7 +351,7 @@ fn no_pending_leak_on_initial_send_failure() {
 	let (route, payment_hash, _, payment_secret) = get_route_and_payment_hash!(nodes[0], nodes[1], 100_000);
 
 	nodes[0].node.peer_disconnected(&nodes[1].node.get_our_node_id(), false);
-	nodes[1].node.peer_disconnected(&nodes[1].node.get_our_node_id(), false);
+	nodes[1].node.peer_disconnected(&nodes[0].node.get_our_node_id(), false);
```

Comment:
Was this a bug in the test? Guessing it didn't matter because the sending side was disconnected?